packages/ember-glimmer/tests/integration/helpers/inline-if-test.js

```diff
@@ -0,0 +1,144 @@
+import { moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import EmberObject from 'ember-runtime/system/object';
+import jQuery from 'ember-views/system/jquery';
+
+import {
+  BASIC_TRUTHY_TESTS,
+  BASIC_FALSY_TESTS,
+  SharedHelperConditionalsTest
+} from '../../utils/shared-conditional-tests';
+
+moduleFor('Helpers test: inline {{if}}', class extends SharedHelperConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{if ${cond} ${truthy} ${falsy}}}`;
+  }
+
+  ['@htmlbars it can omit the falsy argument']() {
+    this.render(`{{if condition 'yes'}}`, { condition: true });
+
+    this.assertText('yes');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('yes');
+
+    this.runTask(() => set(this.context, 'condition', false));
+
+    this.assertText('');
+
+    this.runTask(() => set(this.context, 'condition', true));
+
+    this.assertText('yes');
+  }
+
+  ['@test it can use falsy param as binding']() {
+    this.render(`{{if condition truthy falsy}}`, { condition: false, truthy: 'yes', falsy: 'no' });
+
+    this.assertText('no');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('no');
+
+    this.runTask(() => set(this.context, 'falsy', 'NO!'));
+
+    this.assertText('NO!');
+
+    this.runTask(() => set(this.context, 'falsy', 'no'));
+
+    this.assertText('no');
+  }
+
+  ['@test it respects isTruthy when object changes']() {
+    this.render(`{{if condition 'truthy' 'falsy'}}`, { condition: EmberObject.create({ isTruthy: false }) });
```

Comment:
This is covered by `it tests for `isTruthy` on Ember objects if available` so I'll remove it